conanfile_base.py

```diff
@@ -39,6 +40,18 @@ def _format_ldflags(self, dep):
         return " ".join(libs + lib_paths)
 
     def build(self):
+        def _get_pc_files(package):
+            lib_path = self.deps_cpp_info[package].rootpath
+            for dirpath, _, filenames in os.walk(lib_path):
+                for filename in filenames:
+                    if filename.endswith('.pc'):
+                        shutil.copyfile(os.path.join(dirpath, filename), filename)
+                        tools.replace_prefix_in_pc_file(filename, lib_path)
+            for dep in self.deps_cpp_info[package].public_deps:
+                _get_pc_files(dep)
```

Comment:
Thanks for the PR ! I think these 2 lines can be removed, because the requirements of requirements will already be iterated in the loop below (`self.deps_cpp_info.deps` also has dependencies of dependencies). In the end you can directly inline the content of `_get_pc_files` in the loop over `self.deps_cpp_info.deps`

Reply:
done